### PR TITLE
Update docs for SquashFS images

### PR DESCRIPTION
**Pull request by NeonDaniel**
# Description
Add note in skill installation to use config to ensure skills are re-installed after updates
Add description for the Device Updater Plugin and note root FS changes

# Issues

# Other Notes

### CODE REVIEW

**Review by builderjer — COMMENTED** [review]
The link to OSM community docs is no more. It has not really been updated in quite some time.

File: docs/neon_os/neon_os_updates.md [hunk 1]
```diff
@@ -74,12 +74,47 @@ components is included here.
...
 handles user requests to check for updates. If an update is requested, the skill
-will emit a message to start an update which the [PHAL Plugin](#core-updater-plugin)
-will handle. This skill is also where settings determine if pre-release versions
+will emit a message to start an update which the relevant plugin will handle. 
+This skill is also where settings determine if pre-release versions
 are included in updates.
 
+### Device Updater Plugin (since Neon Core 23.7.31a4)
+NeonOS Releases since July 20, 2023 now use SquashFS to perform a different kind
+of updates. Operating System updates are managed by the
```
Comment (NeonClary) [inline docs/neon_os/neon_os_updates.md:83]: updates --> update

File: docs/neon_os/neon_os_updates.md [hunk 2]
```diff
@@ -74,12 +74,47 @@ components is included here.
...
+process. Note that system packages may be removed during the course of an update,
+so any manually installed packages may need to be re-installed after updating.
+
+#### Advanced Usage
+SquashFS updates attempt to migrate relevant information (like `/var`, `/home`, 
+SSH keys, NetworkManager config, etc.) between updates, but system packages and 
+other manual configuration may be removed as part of the update process. For most
+users, this is helpful to clean up incidental changes and restore an installation
+to a predictable state after updating. For users who want to apply customizations
+that persist updates, here are some guidelines.
```
Comment (NeonClary) [inline docs/neon_os/neon_os_updates.md:102]: persist updates --> persist through updates

File: docs/neon_os/using_neon_os.md [hunk 3]
```diff
@@ -15,4 +15,20 @@ network.
 After network setup is complete, there will be some skills loading in the background;
 when those are loaded, the spinner in the top right corner of the screen will 
 animate and the homescreen will be displayed. At this point, Neon is set up and
-ready to go!
\ No newline at end of file
+ready to go!
+
+## Resetting
+To reset Neon to original settings, there is a menu option under `Settings` ->
+`Factory Settings` -> `Reset Device`. This removes any user settings, restores a
```
Comment (NeonClary) [inline docs/neon_os/using_neon_os.md:22]: 'Reset Device' --> 'Factory Reset'   (the button label is factory reset, not reset device)

**Review by NeonClary — CHANGES_REQUESTED** [review]

**Pull request merged**